src/app/(content)/events/[category]/[id]/page.tsx

```diff
@@ -110,24 +120,44 @@ const Page = () => {
                             </div>
                         </div>
 
-                        <div className="bg-black flex justify-center items-start">
+                        <div className=" flex justify-center items-start">
                             <div>
                                 {/* <h1 className="text-3xl font-bold text-white">{eventInfo?.name}</h1> */}
                                 <Image
                                     src={eventInfo?.thumbnail || ""}
-                                    width={500}
-                                    height={500}
+                                    width={400}
+                                    height={400}
                                     alt="image"
-                                    className="rounded-md shadow-lg shadow-slate-500/50 hover:shadow-slate-500  hover:scale-110  hover:mt-5 transition-all duration-300 w-[500px] mt-16"
+                                    className=" rounded-lg shadow-lg shadow-slate-500/50 mt-16"
                                 />
-                                <div className="mt-8 text-center font-tiara">
+                                <div
+                                    className={cn(
+                                        "tracking-widest  font-medium mt-8 text-center text-xl ",
+                                        tiaraFont.className
+                                    )}
+                                >
                                     <span>
                                         {" "}
-                                        cost ₹ <span className="text-tiara_red">{eventInfo?.costs}</span>{" "}
+                                            cost ₹<span className="text-tiara_red">{eventInfo?.costs}</span><span>/{eventInfo?.team ? "team" : "person"}</span>{" "}
                                     </span>
                                 </div>
                                 <div className="mt-8 text-center">
-                                    <EncryptButton targetText="Register Now" />
+                                    {session && session.user ? (
```

Comment:
You need not check for session here. It will be auto managed by `/register` route. Simply redirect